lib/linguist/languages.yml

```diff
@@ -449,6 +449,14 @@ Cap'n Proto:
   - .capnp
   ace_mode: text
 
+CartoCSS:
+  aliases:
+  - Carto
+  extensions:
+  - .mss
+  ace_mode: text
```

Comment:
I've [tested highlighting of the sample file](https://raw.githubusercontent.com/floscher/openstreetmap-carto/master/addressing.mss) and it looks ok with `CSS`, except for some things:
* //-style comments are not recognized (these are not allowed in CSS but in CartoCSS)
* Strings are not recognized if these are contained in brackets []
* When nesting blocks (marked with curly braces {}), only the first inner block gets marked correctly (see example below)

In this example the word `zoom` and the first `width` are highlighted blue, the other `width` is not:
```
[zoom >= 10] {
  [width <= 2] {
    …
  }
  [width > 2]{
    …
  }
}
```

I can't judge, if these shortcomings are minor enough that setting this to `CSS` would be ok, or if it should be set to `text` until a TextMate-grammar is available.